app/src/main/java/org/oppia/app/utility/KeyboardHelper.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+
+/** KeyboardHelper help to handle the visiblity of softinputkeybord. */
```

Comment:
Nit: ``to change the visibility``. This doesn't really "handle" it like one might expect, but it does let calling code affect the visibility.

Reply:
Done